src/stateToHTML.js

```diff
@@ -341,7 +340,7 @@ class MarkupGenerator {
         }
         return content;
       }).join('');
-      let entity = entityKey ? Entity.get(entityKey) : null;
+      let entity = entityKey ? this.contentState.getEntity(entityKey) : null;
```

Comment:
Correct, Entity.get is deprecated as of 0.10 and replaced by contentState.getEntity